src/main/scala/com/advancedtelematic/campaigner/db/Campaigns.scala

```diff
@@ -336,10 +336,8 @@ protected [db] class CampaignStatusTransition(implicit db: Database, ec: Executi
     for {
       affected <- devicesWithStatus(DeviceStatus.values - DeviceStatus.requested - DeviceStatus.rejected)
       finished <- devicesWithStatus(Set(DeviceStatus.successful, DeviceStatus.failed, DeviceStatus.cancelled))
```

Comment:
Is there a reason we are not using something like `select count(*), status from device_updates where campaign_id=? group by status`?

Reply:
Ditto.